apm-agent-attach-cli/pom.xml

```diff
@@ -42,7 +42,7 @@
         <dependency>
             <groupId>co.elastic.logging</groupId>
             <artifactId>log4j2-ecs-layout</artifactId>
-            <version>${version.log4j2-ecs-layout}</version>
```

Comment:
[for reviewer] using a single version of `ecs-logging-java` (or one of its sub-modules) should be controlled with a single property in `pom.xml`